### PR TITLE
Welcome modal redesign

### PR DESCRIPTION
**Pull request by chrisbendel**

### CODE REVIEW

**Review by nathanstitt — APPROVED** [review]
looks good, just a few comments

File: backend/app/controllers/api/v1/admin/studies_controller.rb [hunk 1]
```diff
@@ -28,6 +28,12 @@ def highlight
     render status: :ok, json: { success: true }
   end
 
+  def welcome
+    Study.where(id: params[:welcome_ids]).update_all(is_welcome: true)
+    Study.where.not(id: params[:welcome_ids]).update_all(is_welcome: false)
```
[the remark this answers is not in the document]
Comment (chrisbendel) [reply backend/app/controllers/api/v1/admin/studies_controller.rb:33]: great catch, I was missing that little detail!

File: backend/app/controllers/api/v1/base_studies_open_api.rb [hunk 2]
```diff
@@ -51,6 +51,11 @@ class Api::V1::BaseStudiesOpenApi
         key :description, 'Is this study highlighted?'
         key :readOnly, true
       end
+      property :is_welcome do
+        key :type, :boolean
+        key :description, 'Is this a welcome study?'
+        key :readOnly, true
+      end
```
[the remark this answers is not in the document]
Comment (chrisbendel) [reply backend/app/controllers/api/v1/base_studies_open_api.rb:58]: the welcome flag is simply for the welcome modal that users will see the first time they come to the site, featured & featured_order dictate which studies show up first in the learning path to the learner as well as which order they will show up in

File: backend/app/controllers/api/v1/participant/studies_controller.rb [hunk 3]
```diff
@@ -76,7 +76,7 @@ def participant_studies
                                                                  :learning_path)
                           .where.not(id: launched_studies.map(&:study_id))
 
-    launched_studies + available_studies
+    available_studies + launched_studies
```
[the remark this answers is not in the document]
Comment (chrisbendel) [reply backend/app/controllers/api/v1/participant/studies_controller.rb:79]: this was a leftover experiment to see if it mattered, oops!

File: backend/app/services/launch_pad.rb [hunk 4]
```diff
@@ -71,6 +73,15 @@ def abort(reason)
...
   attr_reader :user_id
 
+  def notify_if_learning_path_completed
+    return unless @study.learning_path.completed?(@user)
+
+    user_email = UserInfo.for_uuid(@user.id)['email_address']
+    return unless user_email.present? && @study.learning_path.badge_id.present?
+
+    OpenBadgeApi.instance.issue_badge(@study.learning_path.badge_id, ['chrissbendel@gmail.com'])
+  end
```
[the remark this answers is not in the document]
Comment (chrisbendel) [reply backend/app/services/launch_pad.rb:83]: 🤣 🤣 i want all the badges 🤣

**Pull request merged**